public/pages/CreateTransform/containers/CreateTransformForm/CreateTransformForm.tsx

```diff
@@ -503,7 +503,7 @@ export default class CreateTransformForm extends Component<CreateTransformFormPr
           onChangeSourceIndexFilter={this.onChangeSourceIndexFilter}
           onChangeTargetIndex={this.onChangeTargetIndex}
           currentStep={this.state.currentStep}
-          hasAggregation={selectedGroupField.length != 0 || selectedAggregations.length != 0}
+          hasAggregation={selectedGroupField.length != 0 || Object.keys(selectedAggregations).length != 0}
```

Comment:
I think you might want to use `aggList.length` at the moment or append a condition `selectedGroupField.length != 0 || Object.keys(selectedAggregations).length != 0|| aggList.length != 0`  as this is the list I am using for final conversion to the request JSON. Might even remove `selectedGroupField` and `selectedAggregations` in later code cleanup